app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -32,36 +31,27 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({
     validate: zodResolver(userDataValidationSchema),
   });
 
-  async function handleSubmit(values: UserDataValidation) {
-    const { email, name } = values;
+  async function handleSignupSubmit(rawUserData: UserDataValidation) {
+    setFlowState('processing')
 
-    setFlowState('processing');
+    const sanitizedUserData = userDataSanitizationSchema.parse(rawUserData);
```

Comment:
getting an error here because the `userDataSanitizationSchema` import was removed

Reply:
Ooops, must have accidently removed it after merge. Thanks!